src/pages/components/leadspace.mdx

```diff
@@ -70,23 +85,28 @@ For detailed guidelines around usage, also reference the [Lead space pattern pag
 
 There are two color themes to choose from when using Lead space:
 
-- Use `Dark (gray-100)` when the background image has many shadows, is dimly lit, or uses a dark color palette. The gradient overlay will render dark and the text will be white.
-- Use `Light (white)` when the background image is well-lit or brightly colored. The gradient overlay will render white and the text will be gray-100.
+### Dark theme
+
+Use `Dark (gray-100)` when the background image has many shadows, is dimly lit, or uses a dark color palette. The gradient overlay will render dark and the text will be white.
 
-![Dark theme](../../images/pattern/leadspace/LeadSpace_ColorThemes_Dark.jpg)
+![Dark theme](../../images/pattern/leadspace/leadspace_colortheme_dark_demo.jpg)
 
-<Caption>Dark</Caption>
+<Caption>Dark Lead space – Super – 640px height at xlg 1312px breakpoint</Caption>
 <br />
 
-![Light theme](../../images/pattern/leadspace/LeadSpace_ColorThemes_Light.jpg)
+### Light theme
+
+Use `Light (white)` when the background image is well-lit or brightly colored. The gradient overlay will render white and the text will be gray-100.
+
+![Light_theme](../../images/pattern/leadspace/leadspace_colortheme_light_demo.jpg)
```

Comment:
This whole color theme section isn't clear to me. First Carbon doesn't have a light theme and a dark theme. Also, do we do this anywhere else within our components? All of our components are themeable aside like masthead and footer.

Reply:
That's true. It somehow was left hanging. I changed it now to reflect reality.